.coderabbit.yaml

```
@@ -0,0 +1,343 @@
+  # Review components
+  path_filters:
+    # Focus on source code
+    included_paths:
+      - "**.go"
+      - "**.md"
+      - "go.mod"
+      - "go.sum"
+      - "Makefile"
+      - ".github/workflows/**"
+      - "docs/**"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)